modules/core/include/opencv2/core.hpp

```diff
@@ -1697,12 +1697,21 @@ elements.
 CV_EXPORTS_W bool checkRange(InputArray a, bool quiet = true, CV_OUT Point* pos = 0,
                             double minVal = -DBL_MAX, double maxVal = DBL_MAX);
 
-/** @brief converts NaNs to the given number
-@param a input/output matrix (CV_32F type).
+/** @brief Replaces NaNs by given number
+@param a input/output matrix (CV_32F or CV_64F type)
 @param val value to convert the NaNs
 */
 CV_EXPORTS_W void patchNaNs(InputOutputArray a, double val = 0);
 
+/** @brief Generates a mask of finite float values, i.e. not NaNs nor Infs.
+
+An element is set to to 255 (all 1-bits) if all channels are finite.
+@param img Input matrix, should contain float or double elements of 1 to 4 channels
```

Comment:
img => input/src

there is no reason to limit that for images only.

Reply:
Fixed